docs/argo-server-sso.md

```diff
@@ -194,3 +194,21 @@ sso:
 # assuming customClaimGroupName: argo_groups
 workflows.argoproj.io/rbac-rule: "'argo_admins' in groups"
 ```
+
+## Filtering groups
+
+> v3.5 and above
+
+You can configure `filterGroupsRegex` to filter the groups returned by the OIDC provider. Some use-cases for this include:
+
+- You have multiple applications using the same OIDC provider, and you only want to use groups that are relevant to Argo Workflows.
+- You have many groups and exceed the [4KB cookie size limit](https://chromestatus.com/feature/4946713618939904) (cookies are used to store authentication tokens). If this occurs, login will fail.
```

Comment:
Do you happen to know if there's a cookie size limit on any other popular browsers (since the link is particular to Chrome)?

Reply:
Most of them implement the same limit. The link mentions [RFC 6265](https://datatracker.ietf.org/doc/html/rfc6265#section-6.1). The RFC states that 4KB is the _minimum_.

Microsoft mentions something similar in [ASP.NET docs](https://learn.microsoft.com/en-us/previous-versions/ms178194(v=vs.140)?redirectedfrom=MSDN#cookie-limitations).

A lot of websites also refer to this [old tracker](http://browsercookielimits.iain.guru/) -- most there are 4KB, some a few bytes below 4KB, and a few 5KB. All roughly around the same size.

For some reason Chrome is the only one who's implementation is easily accessible in a quick search.

The link is actually from the original issue #10153. We could link to the [IETF RFC](https://datatracker.ietf.org/doc/html/rfc6265#section-6.1) instead, which is less readable, but is more neutral.

I don't have a strong opinion either way

Reply:
Thanks. 

_We could link to the [IETF RFC](https://datatracker.ietf.org/doc/html/rfc6265#section-6.1) instead, which is less readable, but is more neutral. I don't have a strong opinion either way_

I do like readability over neutrality.

Reply:
Yea and Chrome is similarly a high-quality, first-party source (i.e. not a random third-party website)